docs/about/privacy.md

```diff
@@ -0,0 +1,170 @@
+
+If you have any questions about these terms, conditions or the practices of this site or any of the other Services, please contact us at [privacy@CFDE.groups.io](mailto:privacy@CFDE.groups.io).
+
+# Privacy Policy
+
+Last Update Date: 08/30/21
+
+The Common Fund Data Ecosystem Coordination Center (“CFDE-CC” or “we”) has created this Privacy Policy to demonstrate its commitment to privacy and transparency of practices. This Privacy Policy explains the CFDE’s information-gathering and dissemination practices for the CFDE Website and CFDE Portal (the “Services”). By accessing the Services, you consent to the practices and terms of this Privacy Policy. Limited data is collected about your visit to any of our Services in order to help us better understand public use of the sites and to improve and continue to make our Services more useful to visitors. This page describes the information that is automatically collected and stored.
+
+## Information the CFDE Collects
+
+“Personal Information” means any information that can be used to identify you, directly or indirectly, as an individual person. The CFDE collects Personal Information when you visit any of the Services, send CFDE an email, respond to a survey, provide online feedback, register for the CFDE Portal, subscribe to one of the CFDE’s online publications or email newsletters, or otherwise communicate with us. The CFDE also collect information, such as anonymous usage statistics, by using cookies, server logs, and other similar technology as you use the Services.
+
+## Information Provided by You
+
+The CFDE collects Personal Information that you enter through the Services or give to the CFDE in another way. For example:
+
+When you access the Services, the CFDE may collect:
+
+- contact information, such as your name, email address, mailing address, and telephone number
+- employment-related information, such as employer, department, job title and business contact information
+- account information, such as account name, password, security question responses, and a unique identification number
+- product interest information, including favorites and areas of interest
+      
+When you fill out a survey, the CFDE collects, depending on the survey:
+
+- satisfaction with an event or course, product usage, and product satisfaction
+- other information in response to solicited questions, as you voluntarily contribute
+      
+
+## Information the CFDE Obtains from Third Party Sources
+
+The CFDE collects Personal Information about individuals from a number of third party sources.
+
+- Business Partners and Service Providers:  Our business partners and service providers, such as contacts database and website hosting providers, collect Personal Information and share some or all of this information with us.
```

Comment:
@ACharbonneau do you have an example of the Business Partners and Service Providers where we get information like this?